Makefile

```diff
@@ -51,19 +53,22 @@ restart-acp:
 .PHONY: lint
 lint: start-runner
 	docker exec quickstart-runner sh -c "golangci-lint run --fix --deadline=5m ./..."
+	make stop-runner
```

Comment:
I wouldn't like to stop and start container on every `make lint`

Reply:
I did that to avoid container leftovers on pipeline - I can move it into Jenkinsfile.